templates/resources/service_compute.md.tmpl

```diff
@@ -28,11 +28,11 @@ The `package` block supports uploading or modifying Wasm packages for use in a F
 
 ## Product Enablement
 
-The [Product Enablement](https://developer.fastly.com/reference/api/products/enablement/) APIs allow customers to enable and disable specific products.
+The [Product Enablement](https://developer.fastly.com/reference/api/products/) APIs allow customers to enable and disable specific products.
 
 Not all customers are entitled to use these endpoints and so care needs to be given when configuring a `product_enablement` block in your Terraform configuration.
 
-Consult the [Product Enablement Guide](/docs/guides/product_enablement) to understand the internal workings for the `product_enablement` block.
+Consult the [Product Enablement Guide](../guides/product_enablement) to understand the internal workings for the `product_enablement` block.
```

Comment:
In my testing using the developer tools in a browser and modifying the link target, a relative target worked fine... as long as the rendered pages end up in the same paths as the source files, this should be safe.